src/org/loklak/harvester/YoutubeScraper.java

```diff
@@ -205,9 +203,14 @@ public static JSONObject parseVideo(final BufferedReader br) throws IOException
         return json;
     }
     
-    private static long parseNumber(String n) throws NumberFormatException {
-        String withoutSpaces = CharMatcher.WHITESPACE.removeFrom(n);
-        return Long.parseLong(withoutSpaces);
+    private static String parseValue(String n) {
+        String number_value = CharMatcher.WHITESPACE.removeFrom(n);
+        Pattern number_format = Pattern.compile("((\\d)\\d{0,2})(,?\\d{3})*(\\.[\\d]+)?([MBK]?)");
```

Comment:
@singhpratyush if we set it as private static member, it will consume memory for complete lifecycle of the YoutubeScraper oubject. It is used only in 1 method, So maybe local declaration suits it